packages/teleport/src/components/SelectFilters/Pager.tsx

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright 2021 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import Icon, { CircleArrowLeft, CircleArrowRight } from 'design/Icon';
+import { Text, Flex } from 'design';
+
+export default function Pager({
+  startFrom = 0,
+  endAt = 0,
+  totalRows = 0,
+  onPrev,
+  onNext,
+}: Props) {
+  const isPrevDisabled = totalRows === 0 || startFrom === 0;
+  const isNextDisabled = totalRows === 0 || endAt === totalRows;
+  const initialStartFrom = totalRows > 0 ? startFrom + 1 : 0;
+
+  return (
+    <Flex m={2} justifyContent="flex-end">
+      <Flex alignItems="center" ml={2}>
+        <Text typography="body2" style={{ color: '#4b4b4b' }}>
```

Comment:
I found this color in a few places, maybe it's worth to extract it to theme?

Reply:
it's kind of specific color to the SelectFilter component, so what I did was remove it from here, and set its parent container color, so at least its contained in one file